test/cljs_time/coerce_test.cljs

```diff
@@ -81,7 +81,9 @@
   (is (= 893462400 (to-epoch (js/Date. 893462400000))))
   (is (= (long 0) (to-epoch 0)))
   (is (= 893462400 (to-epoch 893462400000)))
-  (is (= 893462400 (to-epoch "1998-04-25T00:00:00.000Z"))))
+  (is (= 893462400 (to-epoch "1998-04-25T00:00:00.000Z")))
+  (testing "returns an int"
```

Comment:
@danielcompton I added two rounding examples using long ints, ditching the string date since I believe the `"1998-04-25T00:00:00.000Z"` example is there to show that strings can be accepted as inputs and I didn't want to group it with our rounding tests here.